UefiCpuPkg/PiSmmCpuDxeSmm/X64/PageTbl.c

```diff
@@ -1086,8 +1086,10 @@ SmiPFHandler (
       DEBUG_CODE (
         DumpModuleInfoByIp ((UINTN)SystemContext.SystemContextX64->Rip);
       );
-      CpuDeadLoop ();
-      goto Exit;
+      // MSCHANGE - Allow system to reset instead of halt in test mode.

```

Comment:
MU_CHANGE

Reply:
Throughout the file, or just this one?

Reply:
Just this one. There's a bunch of legacy ones that we haven't changed yet, but we're trying to stay consistent going forwards.

Reply:
Done.